sbol2/object.py

```diff
@@ -46,7 +47,7 @@ def _compare_owned_objects(obj1, obj2):
     return True
 
 
-class SBOLObject:
+class SBOLObject():
```

Comment:
Why do we need these parentheses? Since we're not extending an object we shouldn't need them in Python 3.

Reply:
Who put that there?  I'll remove that.